db/move_items.py

```diff
@@ -1,5 +1,11 @@
 #!/bin/env python
 #from __future__ import print_function
+"""
+Copyright (c) 2018.
+This file is part of globocom/zabbix-scripts
+(see https://github.com/globocom/zabbix-scripts).
+License: 3-clause BSD, see https://opensource.org/licenses/BSD-3-Clause
+"""
 import pymysql,sys,os,json
```

Comment:
Copyright notice should include (year, Name + email/site), such as:
```
Copyright (c) 2018, Globo.com <https://github.com/globocom>
```

For python code, we usually use this syntax for Copyright notice:
```
# Copyright (c) 2018, Globo.com <https://github.com/globocom>
# This file is part of globocom/zabbix-scripts
# (see https://github.com/globocom/zabbix-scripts).
# License: 3-clause BSD, see https://opensource.org/licenses/BSD-3-Clause
```

Reply:
Thanks for the guidance. I've already made the changes. :+1: